src/ocean/core/BitArray.d

```diff
@@ -86,11 +86,13 @@ struct BitArray
                 uint[] buf = ptr[0 .. olddim];
 
                 buf.length = newdim; // realloc
+                enableStomping(buf);
+
                 ptr = buf.ptr;
-                if( newdim & 31 )
+                if( auto pad_bits = newlen & 31 )
```

Comment:
I am not sure if adding parenthesis would make it better
```D
if( auto pad_bits = (newlen & 31) )
```

Otherwise I could revert it to the original version and just replace `newdim` to `newlen` but then we will have to compute `newlen & 31` twice.